config.yaml

```diff
@@ -127,6 +127,8 @@ projects:
       hhu:
         contact: "Ute Fischer"
         project_id: 2
+      genepanels:
```

Comment:
the syntax depends on indentation! You need to remove two whitespaces to bring it to the same level as `snupy` and not below `snupy` as it is now

Reply:
done

Reply:
:+1: